distributed/tests/test_active_memory_manager.py

```diff
@@ -580,7 +584,7 @@ async def _tensordot_stress(c):
 
 
 @pytest.mark.slow
-@pytest.mark.xfail(reason="https://github.com/dask/distributed/issues/5371")
+@pytest.mark.avoid_ci(reason="distributed#5371")
```

Comment:
Just a subjective comment, but I find using the full hyperlink in these situations nicer because I can quickly copy and paste the link into a browser and go right to the issue.

Reply:
Also, nice usage of `avoid_ci` 👍